apps/pyramid_app.py

```diff
@@ -16,10 +16,10 @@ def index(request):
     config.add_view(index, route_name="index")
 
     if os.environ.get("VULNPY_USE_CONTRAST"):
-        # TODO: be able to import as "contrast.pyramid.ContrastMiddleware"
-        config.add_tween(
-            "contrast.agent.middlewares.pyramid_middleware.PyramidMiddleware"
-        )
+        from contrast.pyramid import ContrastMiddleware
+
+        app = config.make_wsgi_app()
+        app = ContrastMiddleware(app)
 
     app = config.make_wsgi_app()
```

Comment:
I see that this worked, but honestly I'm not totally sure why 😅 

Can you call `app = config.make_wsgi_app()` only once, before the `if`?